src/main/java/org/snomed/snowstorm/core/data/services/CodeSystemService.java

```diff
@@ -310,6 +310,11 @@ private void copyDetailsFromCacheEntry(CodeSystem codeSystem, Pair<Date, CodeSys
 		codeSystem.setLanguages(cachedCodeSystem.getLanguages());
 		codeSystem.setModules(cachedCodeSystem.getModules());
 		codeSystem.setDependantVersionEffectiveTime(cachedCodeSystem.getDependantVersionEffectiveTime());
+		CodeSystemVersion latestVersion = codeSystem.getLatestVersion();
+		CodeSystemVersion cachedLatestVersion = cachedCodeSystem.getLatestVersion();
+		if (latestVersion != null && cachedLatestVersion != null) {
+			latestVersion.setDependantVersionEffectiveTime(cachedLatestVersion.getDependantVersionEffectiveTime());
+		}
```

Comment:
When requesting Code Systems, the data is read from the cache. This block simply maps the cache to the response.